src/App.tsx

```diff
@@ -1,26 +1,88 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { Header } from './components/Header/Header';
+import { getVisibleTodos } from './utils/getVisibleTodos';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { TodoItem } from './components/TodoItem/TodoItem';
+import { ErrorNotification } from './components/Errors/Errors';
+import { ErrorMessages } from './types/ErrorMessages';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState(Status.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadingId, setLoadingId] = useState<number[]>([]);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => setErrorMessage(ErrorMessages.UnableLoadTodos))
+      .finally(() => {
+        setTimeout(() => {
+          setErrorMessage('');
+        }, 3000);
```

Comment:
You should remove the error only if you have it.
You can create a handler to set and remove error so you don't have to do it manually in each API call
```typescript
const handleError = (error: ErrorMessages) => {
  setErrorMessage(error);
  setTimeout(() => {
    setErrorMessage('');
  }, 3000);
};
```